### PR TITLE
Pureinsights blog post and author bio

### PR DESCRIPTION
**Pull request by krisfreedain**
### Description
New blog post from Pure Insights & author bio
 
### Issues Resolved
N/A

### Check List
- [ X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2022-04-15-Partner-Highlight-Pureinsights-adds-OpenSearch-to-its-Discovery-Platform.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Partner Highlight: Pureinsights adds OpenSearch to its Discovery Platform"
+authors:
+  - plewis
+date: 2022-04-15 08:50:01
+categories:
+  - partners
+  - technical-post
```
Comment (krisfreedain) [inline _posts/2022-04-15-Partner-Highlight-Pureinsights-adds-OpenSearch-to-its-Discovery-Platform.markdown:9]: Just caught this - please remove "-technical-post" as it causes issues with the Jekyll URL

File: _posts/2022-04-15-Partner-Highlight-Pureinsights-adds-OpenSearch-to-its-Discovery-Platform.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Partner Highlight: Pureinsights adds OpenSearch to its Discovery Platform"
+authors:
+  - plewis
+date: 2022-04-15 08:50:01
```
Comment (krisfreedain) [inline _posts/2022-04-15-Partner-Highlight-Pureinsights-adds-OpenSearch-to-its-Discovery-Platform.markdown:6]: Please do a quick update to the date to 2022-04-22

thanks

**Review by krisfreedain — COMMENTED** [review]
@nereo-campos-pi -- noticed 2 small things in staging - can you make these 2 minor changes? thank you

File: _posts/2022-04-15-Partner-Highlight-Pureinsights-adds-OpenSearch-to-its-Discovery-Platform.markdown (hunk 1 above)
Comment (nereo-campos-pi) [reply _posts/2022-04-15-Partner-Highlight-Pureinsights-adds-OpenSearch-to-its-Discovery-Platform.markdown:9]: PR-775 has the required changes

File: _posts/2022-04-15-Partner-Highlight-Pureinsights-adds-OpenSearch-to-its-Discovery-Platform.markdown (hunk 2 above)
Comment (nereo-campos-pi) [reply _posts/2022-04-15-Partner-Highlight-Pureinsights-adds-OpenSearch-to-its-Discovery-Platform.markdown:6]: PR-775 has the required changes 
Thank you

**Pull request merged**